test/FsAutoComplete.Tests.Lsp/CodeFixTests.fs

```diff
@@ -1268,6 +1282,391 @@ let private removeUnusedOpensTests state =
         selectCodeFix
   ])
 
+let private renameParamToMatchSignatureTests state =
+  let selectCodeFix expectedName = CodeFix.withTitle (RenameParamToMatchSignature.title expectedName)
+
+  // requires `fsi` and corresponding `fs` file (and a project!)
```

Comment:
you could potentially generate this at test-time to a random directory instead of being forced to check in empty data?

Reply:
The "complex" part here is the project file. 
Which of course can be written to file too -- but now it's either content of fsproj in tests or adjusting a freshly created project (with `dotnet new ...` in some dir).  
And would require getting a dir, and then writing all necessary files....but that adds additional complexity


We're now way more complex than just loading an existing project -- even when just project just contains empty "dummy" files

Reply:
Not a need for now, but I've been looking at [this library](https://github.com/jeffkl/MSBuildProjectCreator) for defining project files in code